test/e2e/e2e_test.go

```diff
@@ -992,6 +993,17 @@ func deletePodAndPVCs(config *localTestConfig, pod *v1.Pod) error {
 	return nil
 }
 
+func init() {
+	// Register framework flags, then handle flags and Viper config.
+	framework.RegisterCommonFlags()
+	framework.RegisterClusterFlags()
+	framework.AfterReadingAllFlags(&framework.TestContext)
+}
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+}
```

Comment:
prow base image is upgraded to use go1.13.4, there is a breaking change in go1.13 testing package (xref: https://golang.org/doc/go1.13#testing) that testing flags are not registered in the initialization phase, so we can't prase the flags in init() now.